client/src/components/CheckboxGrid.tsx

```diff
@@ -84,7 +90,10 @@ class CheckboxRow extends Component<CheckboxRowProps> {
 						<div className="checkbox">
 							<input
 								type="checkbox"
-								className={props.client.highlightedIndex === idx ? "highlighted" : undefined}
+								className={
+                                    `${checkboxStyle} ${isHighlighted(idx)} ${tickMarkToggle}`
+                                    .split(" ").filter( x => x !== '' ).join(' ')
```

Comment:
This line seems unnecessary

Reply:
It is unnecessary. I probably added that because there was odd space between the classes. No real purpose.